specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json

```diff
@@ -1762,16 +1824,68 @@
           "$ref": "#/definitions/ManagedClusterAADProfile",
           "description": "Profile of Azure Active Directory configuration."
         },
-        "apiServerAuthorizedIPRanges": {
+        "privateLinkResources": {
           "type": "array",
           "items": {
-            "type": "string"
+            "$ref": "#/definitions/ManagedClusterPrivateLinkResource"
           },
-          "description": "(PREVIEW) Authorized IP Ranges to kubernetes API server."
+          "description": "Properties related to private link cluster."
+        },
+        "apiServerAccessProfile": {
+          "$ref": "#/definitions/ManagedClusterAPIServerAccessProfile",
+          "description": "Access profile for managed cluster API server."
         }
       },
       "description": "Properties of the managed cluster."
     },
+    "ManagedClusterPrivateLinkResource": {
```

Comment:
Is there a benefit of modeling this as a child resource type under managedClusters and have GET APIs. Similar to how SQL is doing it - https://github.com/Azure/azure-rest-api-specs/pull/6866/

Reply:
@weinong  can you comment here?

Reply:
we decided to remove this in this version.